pkgs/applications/networking/instant-messengers/cinny-desktop/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv
+, lib
+, dpkg
+, fetchurl
+, autoPatchelfHook
+, openssl
+, gnused
+, webkitgtk
+, makeWrapper
+, disableCompositing ? false # https://bugs.webkit.org/show_bug.cgi?id=180739
+}:
+
+stdenv.mkDerivation rec {
+
+  name = "cinny-desktop";
+  version = "2.0.4";
+
+  src = fetchurl {
+    url = "https://github.com/cinnyapp/cinny-desktop/releases/download/v${version}/cinny_${version}_amd64.deb";
+    sha256 = "sha256-udjLlYZTqOBTF9H7N6wJnIomFAeQvAczmEp3imE2IGY=";
+  };
+
+  nativeBuildInputs =
+    [ autoPatchelfHook dpkg gnused ]
+    ++ lib.optionals disableCompositing [ makeWrapper ];
+
+  buildInputs = [
+    openssl
+    webkitgtk
+  ];
+
+  unpackPhase = "true";
+
+  installPhase = ''
+    mkdir -p $out/bin $out/share
+    dpkg -x $src $out/tmp
```

Comment:
This shouldn't be done at install phase. You can use `unpackCmd` instead.

Reply:
Also why is this first copied into tmp and later moved? We can copy this straight into place and change everything we want or delete unwanted files before copying